src/handlers/ens.ts

```diff
@@ -489,12 +488,21 @@ export const fetchAccountPrimary = async (accountAddress: string) => {
 };
 
 export function prefetchENSIntroData() {
-  for (const name of ensIntroMarqueeNames) {
-    prefetchENSAddress({ name }, { staleTime: Infinity });
-    prefetchENSAvatar(name, { cacheFirst: true });
-    prefetchENSCover(name, { cacheFirst: true });
-    prefetchENSRecords(name, { cacheFirst: true });
-    prefetchFirstTransactionTimestamp({ addressOrName: name });
+  const ensMarqueeQueryData = queryClient.getQueryData<{
+    ensMarquee: EnsMarqueeAccount[];
+  }>([ENS_MARQUEE_QUERY_KEY]);
```

Comment:
Just double checking: if we haven't already fetched this query key, this will return undefined. Cool?

Reply:
falling back no ty